conman/routes/utils.py

```diff
@@ -13,7 +13,7 @@ def split_path(path):
     path = path.rstrip('/')
 
     while path:
-        paths.append(path + '/')
+        paths.insert(1, path + '/')
         path = os.path.split(path)[0]
         if path == '/':
             break
```

Comment:
It probably doesn't matter much, but if we wanted to optimise this, we could try using a `collections.deque` and the `appendleft` method. The advantages might not appear with typical length urls though, so profiling would be a good idea.